stacker/actions/build.py

```diff
@@ -28,6 +28,22 @@ def should_update(stack):
     return True
 
 
+def should_submit(stack):
+    """Tests whether a stack should be submitted to CF for update/create
+
+    Args:
+        stack (stacker.stack.Stack): The stack object to check.
```

Comment:
Can you use the syntax found here for the class - it looks better in the generated docs: https://github.com/remind101/stacker/blob/master/stacker/stack.py#L50